charts/app-config/templates/env-configmap.yaml

```
@@ -16,6 +16,7 @@ data:
   GOVUK_APP_DOMAIN_EXTERNAL: {{ .Values.publishingDomainSuffix }}
   GOVUK_ASSET_ROOT: https://{{ .Values.assetsDomain }}
   GOVUK_CSP_REPORT_URI: {{ .Values.cspReportURI | quote }}
+  GOVUK_ENABLE_SEARCH_AUTOCOMPLETE: {{ .Values.enableSearchAutocomplete }}
```

```suggestion
  {{- if .Values.enableSearchAutocomplete }}
  GOVUK_ENABLE_SEARCH_AUTOCOMPLETE: "true"
  {{- end }} 
```

Otherwise I believe that even if we remove the `enableSearchAutocomplete` declaration an env var will still be created so we won't be able to do a simple check of `if ENV["GOVUK_ENABLE_SEARCH_AUTOCOMPLETE"]` and instead need to check the contents of the env var - which given there's a bunch of apps could be a bit tedious (and relates to the eternal struggle of configuring the boolean env var).


(slight caveat that I'm not 100% sure of exactly helm works in this regard but expect a configmap of "GOVUK_ENABLE_SEARCH_AUTOCOMPLETE:` produces `ENV["GOVUK_ENABLE_SEARCH_AUTOCOMPLETE"] = ""`)

---

I'm not sure about booleans either. I wasn't sure if boolean would be preserved.

Regarding the name of the ENV VAR, I made it "truthy" because that hurts my brain less when it came to using the env var in the rendering apps.

This change was just trying to figure out how it "could" be done, so I'm happy for you to take over and flip this.